assets/js/pages/ClusterDetails/HanaClusterDetails.test.jsx

```diff
@@ -385,10 +388,20 @@ describe('HanaClusterDetails component', () => {
     expect(screen.getByText('Attributes')).toBeInTheDocument();
     expect(screen.getByText('Resources')).toBeInTheDocument();
 
-    Object.keys(resources[0]).forEach((key) => {
-      expect(screen.getByText(key)).toBeInTheDocument();
-      screen.getAllByText(resources[0][key]).forEach((element) => {
-        expect(element).toBeInTheDocument();
+    tableColumns.forEach(({ key, title }) => {
```

Comment:
I had to refactor this test as we use now a custom config, previously we were iterating through the resource key value pairs and used it to render our table.  Now we provide [a config](https://github.com/trento-project/web/pull/2366/files#diff-e440c92bcaacad26fb9ca18f7a4e0186e6af4712f0eaee988ec4c1414dd0b657R9), so it is more  explicit. 

Making this change, effects this part a lot. I tried to refactor it and adjust to the new config. 
not fully happy with it, as it is quite long!

Open for refactoring suggestions.

Reply:
Hardcode the tests in a `it.each` please. For each column. And the same as before. This could go in the `AttributeResources.test.jsx`, and remove this from here